### ISSUE TITLE
[FEATURE] Delete page properties

### ISSUE THREAD

**Issue by Berdir** [mariosimao/notion-sdk-php#257]
**Is your feature request related to a problem? Please describe.**

First I thought being able to delete properties is only missing as a dedicated PHP method, but even doing the following does nothing, so I guess properties that aren't being sent are not removed:

```
if ($delete_properties) {
      $all_properties = $instance_page->properties;
      $all_properties = array_diff_key($all_properties, array_flip($delete_properties));
      $instance_page = $instance_page->changeProperties($all_properties);
    }
```

Setting the value to empty apparently also doesn't work, resulting in the following errors:

```
Uncaught PHP Exception Notion\Exceptions\ApiException: "body failed validation. Fix one:
body.properties.NAME.url should be populated or `null`, instead was `""`.
```

Nor is it possible to pass NULL as the value, as it is type hinted on string.

**Describe the solution you'd like**

Support NULL values? add removeProperty() methods? Not sure how it needs to work on the API level against Notiion.

**Comment by mariosimao** [comment]
Hi @Berdir, sorry for the late response. The property you are trying to delete is of type URL? If yes, indeed there is no possible way to delete yet.

**Comment by mariosimao** [comment]
@Berdir I have just released [v1.10](https://github.com/mariosimao/notion-sdk-php/releases/tag/v1.10.0). It includes new methods for creating and updating page properties with empty values.

There are now two possible ways to clear the value of a property:

1. Replace the property by an empty one.

```php
$page = $notion
    ->pages()
    ->find($pageId)
    ->addProperty("Property name", Url::createEmpty());

$notion->pages()->update($page);
```

2. Or get the property, clear it and update.

```php
$page = $notion->pages()->find($pageId);

$property = $page->properties()->getUrl("Property name");
$property = $property->clear();

$page = $page->addProperty("Property name", $property);

$notion->pages()->update($page);
```

**Comment by Berdir** [comment]
Hi. Thanks, I'd consider that very fast feedback, not slow. Will test this.
